Purchases/Networking/Operations/PostReceiptDataOperation.swift

```diff
@@ -105,3 +73,41 @@ class PostReceiptDataOperation: CacheableNetworkOperation {
     }
 
 }
+
+extension PostReceiptDataOperation.PostData: Encodable {
+
+    private enum CodingKeys: String, CodingKey {
+
+        case fetchToken
+        case appUserID
+        case isRestore
+        case observerMode
+        case attributes
+        case presentedOfferingIdentifier
+
+    }
+
+    func encode(to encoder: Encoder) throws {
```

Comment:
what's the behavior when encoding throws?

Reply:
Same as before, `HTTPClient` fails to begin the request.

Reply:
See `BackendTests.testHandlesInvalidJSON` for example.